//test error

int main() {
  int foo() {
    return 3;
  }
  return foo();
}
